Sources/TuistAcceptanceTesting/FixtureTestingTrait.swift

```diff
@@ -0,0 +1,103 @@
+import ArgumentParser
+import FileSystem
+import Foundation
+import Path
+import Testing
+import TuistSupport
+import TuistSupportTesting
+@testable import TuistKit
+
+public struct FixtureTestingTrait: TestTrait, SuiteTrait, TestScoping {
+    let fixturePath: AbsolutePath
+
+    init(fixture: String, fixturesDirectory: AbsolutePath = Self.defaultFixturesDirectory()) {
+        fixturePath = fixturesDirectory.appending(component: fixture)
+    }
+
+    public static func defaultFixturesDirectory() -> AbsolutePath {
+        // swiftlint:disable:next force_try
+        return try! AbsolutePath(
+            validating: ProcessInfo.processInfo.environment[
+                "TUIST_CONFIG_SRCROOT"
+            ]!
+        ).appending(component: "fixtures")
```

Comment:
I wonder if instead of using `TUIST_CONFIG_SRCROOT`, we can use this file's location? Do you remember why we use `TUIST_CONFIG_SRCROOT` (I don't)